documentation/API-design-guidelines.md

```diff
@@ -631,7 +634,10 @@ An error representation must not be different from the representation of any res
 - A field "`status`", which can be identified in the response as a standard code from a list of Hypertext Transfer Protocol (HTTP) response status codes.
 - A unique error "`code`", which can be identified and traced for more details. It must be human readable; therefore, it must not be a numeric code. In turn, to achieve a better location of the error, you can reference the value or field that is causing it, and include it in the message. 
 - A detailed description of "`message`"
-  
+
+All these aforementioned fields are mandatory in Error Responses.
+`status` and `code` fields have normative nature, so as their use has to be normalize (see [Section 6.2](##62-normalization-use-of-error-responses)). On the other hand, `message` is informative and within this document an example is shown.
```

Comment:
I think 'has to be normalized' or 'has to be standardized' would be better

Reply:
changed